docs/file-structure.md

```diff
@@ -26,6 +26,18 @@ This directory should be placed in the plugin's root, next to the `public.php` f
     - https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/invoice-csv-export/src/public
     - https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/plugins/invoice-csv-export/src/templates/form.php#L10
 
+### hook_*.php files
+*Available since UCRM 2.16.0-beta1.*
+When the plugin is installed, updated, configured, enabled, disabled or removed, UCRM will automatically call the respective hook file which you can use to execute any custom commands, e.g. run initial config queries, run migrations or setup database etc.
```

Comment:
```suggestion
When the plugin is installed, updated, configured, enabled, disabled or removed, UCRM will automatically call the respective hook file (if it exists). You can use this to execute any custom commands, e.g. run initial config queries, run migrations or setup database etc.
```

Reply:
"After" doesn't feel correct. You can't run a hook **after** the plugin is removed.

Reply:
But for all other events it is after. Only the remove is basically in the middle.